graphql_compiler/schema_generation/orientdb/schema_graph_builder.py

```diff
@@ -437,35 +437,7 @@ def _try_get_base_connections(class_name, inheritance_structure, links, abstract
     )
 
 
-def _link_schema_elements(elements, inheritance_structure):
-    """For each edge, link the schema elements it connects to each other."""
-    _, edge_class_names, _ = _get_vertex_edge_and_non_graph_class_names(inheritance_structure)
-
-    for edge_class_name in edge_class_names:
-        edge_element = elements[edge_class_name]
-
-        from_class_name = edge_element.base_in_connection
-        to_class_name = edge_element.base_out_connection
-
-        if not from_class_name or not to_class_name:
-            continue
-
-        edge_schema_element = elements[edge_class_name]
-
-        # Link from_class_name with edge_class_name
-        for from_class in inheritance_structure.subclass_sets[from_class_name]:
-            from_schema_element = elements[from_class]
-            from_schema_element.out_connections.add(edge_class_name)
-            edge_schema_element.in_connections.add(from_class)
-
-        # Link edge_class_name with to_class_name
-        for to_class in inheritance_structure.subclass_sets[to_class_name]:
-            to_schema_element = elements[to_class]
-            edge_schema_element.out_connections.add(to_class)
-            to_schema_element.in_connections.add(edge_class_name)
-
-
-def _get_graphql_rep_of_non_graph_elements(non_graph_elements, inheritance_structure):
+def _get_graphql_representation_of_non_graph_elements(non_graph_elements, inheritance_structure):
```

Comment:
This nit is out of the scope of the pull request, but it doesn't hurt to add it.